api-inference-community/docker_images/flair/app/main.py

```diff
@@ -56,13 +60,12 @@ async def startup_event():
     handler.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(message)s"))
     logger.handlers = [handler]
 
-    task = os.environ["TASK"]
-    model_id = os.environ["MODEL_ID"]
-
-    app.pipeline = get_pipeline(task, model_id)
+    app.get_pipeline = get_pipeline
```

Comment:
Does this need this below? (just as in #123) 

```
try:
        get_pipeline()
    except Exception:
        # We can fail so we can show exception later.
        pass
```

Reply:
It will help usage a bit, not a strict requirement.

With this, starlette will load the model form the start, without it it will load it when first request hits.
(It's only memory load, it will already be on disk because of `prestart.sh`)